vnext/local-cli/generator-windows/index.js

```diff
@@ -116,8 +116,8 @@ function copyProjectTemplateAndReplace(
   copyAndReplaceAll(path.join(srcPath, 'assets'), destPath, path.join(windowsDir, newProjectName, 'Assets'), templateVars, options.overwrite);
   copyAndReplaceAll(path.join(srcPath, 'src'), destPath, path.join(windowsDir, newProjectName), templateVars, options.overwrite);
 
-  console.log(chalk.white.bold('To run your app on UWP:'));
-  console.log(chalk.white('   react-native run-windows'));
+  console.log(chalk.white.bold('To run your app on Windows:'));
+  console.log(chalk.white('   npx react-native run-windows'));
 }
```

Comment:
this _shouldn't_ be necessary - when you install the react-native package globally it should create a .cmd file in a folder in your appdata that is in your PATH env var.
Do you run into this if you npm install **-g** react-native ?

Reply:
Installing react-native globally (or most npm packages) is no longer recommended.  It gets out of date, and there is nothing keeping the version you have globally in sync with what you have in your local package.  It is also impossible to have multiple react-native apps on different versions if you are relying on global install state.

Reply:
good to know :) we should update the docs to reflect this